### ISSUE TITLE
Multiple tabs start over my session again and again

### ISSUE THREAD

**Issue by BigAndini** [zammad/zammad#2306]
### Infos:

* Used Zammad version: zammad-2.6.0-1539902613.d89d86d8.centos7.x86_64
* Installation method (source, package, ..): yum repo: https://dl.packager.io/srv/rpm/zammad/zammad/stable/el/7/$basearch
* Operating system: CentOS 7.5.1804
* Database + version: postgresql-server-9.2.24-1.el7_5.x86_64
* Elasticsearch version: elasticsearch-5.6.12-1.noarch
* Browser + version: Firefox 62.0.3 (64-Bit)

### Expected behavior:

* When working with multiple tabs, be able to switch fast between tabs to get tickets solved quickly.

### Actual behavior:

* When multiple tabs are opened zammad is only able to have the active session in the last tab. All other tabs have a modal that states the question to restart the session.

### Steps to reproduce the behavior:

* I just need to open zammad, go to the overview page, open multiple tickets in tabs and there the message appears.

![grafik](https://user-images.githubusercontent.com/8452304/47340548-e762f880-d69e-11e8-9295-0e8673749958.png)

The message is in german, because my system is configured to a german frontend. It states:

Session

A new session was created for your user. To prevent conflicts in colletive opened tasks, this session has been stopped.

This is my translation, maybe the translation of zammad itself can be different.

Yes I'm sure this is a bug and no feature request or a general question. I'm pretty sure that such a great tool as zammad should support multitabbing and I hope I'm correct in this platform and it is a bug and not just a configuration issue.

**Comment by thorsteneckel** [comment]
Hi @inbaz - thanks for creating an issue. However, this works as designed. Zammad is a Single Page Application that stores the current state of your session (which Tab is open, Article drafts etc.) which can't be shared across multiple tabs (that easy). So this is not a bug but we'd like to hear your feedback anyway on what's your use case for which you need a second tab. Please refer to the [community board](http://community.zammad.org/) for posting the feedback. On the issue tracker over 100 people will receive the mail which will create a lot of noise. Thanks for your understanding. Closing now.

**Comment by martinvonwittich** [comment]
We're currently evaluating Zammad, and we've also stumbled over this. Generally I'm certain that our agents will get used to this (after all WhatsApp Web works the same way), but there is one situation where this is annoying: when sharing links to tickets.

Scenario:
- I have opened a ticket and I want to tell another agent about this ticket.
- I send a link to that ticket (e.g. https://zammad.example.com/#ticket/zoom/12) to the other agent via our internal company chat.
- The other agent opens the link, and the browser will of course open this in a new tab.
- When the other agent has finished looking at the ticket, he might close the tab.
- But now the first tab shows the "This session will be stopped to prevent a conflict" message, and clicking on the button causes Zammad to reload, which takes at least a second is a bit annoying.

Maybe there's a solution possible where the new Zammad tab will just cause the first Zammad tab to open the ticket? Or the "This session will be stopped to prevent a conflict" message in the first Zammad tab automatically vanishes after the second tab has been closed?
